src/jit/compiler.h

```diff
@@ -3099,6 +3101,18 @@ class Compiler
     bool compSupportsHWIntrinsic(InstructionSet isa);
     bool isScalarISA(InstructionSet isa);
 #endif // _TARGET_XARCH_
+#ifdef _TARGET_ARM64_
+    NamedIntrinsic lookupHWIntrinsic(const char* className, const char* methodName);
+    GenTree* impHWIntrinsic(NamedIntrinsic        intrinsic,
+                            CORINFO_METHOD_HANDLE method,
+                            CORINFO_SIG_INFO*     sig,
+                            bool                  mustExpand);
+    GenTree* impUnsupportedHWIntrinsic(unsigned              helper,
```

Comment:
The signature looks identical between the two, can we share it?

Reply:
It should move to a shared file, but I was goiing to make that a separate PR.  So it could be reviewed faster.